README.md

```diff
@@ -1,8 +1,134 @@
-# duckbase
+duckbase
+=========================
 React + Redux Wrapper for Firebase
 
 ![Duck Taunt](ducktaunt.jpg)
 
-### License
+## Features
+- Automatic binding to the data your components need
+- Automatic unbinding for data no longer needed
+- Mirror your firebase data in redux without the boilerplate
+- Supports Firebase queries
+- First class support for Typescript
+
+## Installation
+To install the latest version:
+```
+yarn add duckbase
+```
+or
+```
+npm install --save duckbase
+```
+
+## Example
+
```

Comment:
Should probably explain the example setup before showing code. For example ...

"Here is a todo app with a calendar integration. The `CalendarReducer` handles the local state for selecting the calendar date range, and the firebase reducer will keep track of mirroring your firebase data.

"First, wrap your component with the `FirebaseProvider` and add the `firebaseReducer` to your `rootReducer`."

And then before TodoCalendar.js, you could have ...

"This component introduces duckbase two ways: one with a normal fetch and the other with a fancy query."

And then break out the important pieces from this into chunks to explain the `getValue` and `getQueryValue` and how you form those by passing the path into the `firebaseConnect`